rpc/src/v1/types/account_info.rs

```diff
@@ -41,3 +44,19 @@ pub struct HwAccountInfo {
 	/// Device manufacturer.
 	pub manufacturer: String,
 }
+
+/// account derived from a signature
+/// as well as information that tells if it is valid for
+/// the current chain
+#[derive(Debug, Clone, Serialize)]
+#[serde(rename_all="camelCase")]
+pub struct BasicAccount {
```

Comment:
Why not rename this to `RecoveredAccount` instead of `BasicAccount` which can be confused with `ethcore::BasicAccount` (state of an account) ?